assets/css/v2/lcd_common/departures/header.scss

```diff
@@ -35,8 +35,17 @@
   margin-left: 32px;
   font-size: 40px;
   line-height: 54px;
+  padding-bottom: 44px;
 
   strong {
     @include fonts.inter--bold;
   }
 }
+
+.departures-header-container + .departure-row::before {
```

Comment:
This seems like it would add the divider even when there is no subtitle (and in that case would result in some unwanted doubling-up on borders, since there is also a bottom border on the header). Do the header and subtitle actually need to be in a common container element? If not, i.e. if the element tree could look like:

* header
* subtitle
* departure
* departure
* ...

Then we could use a selector like `.departures-subtitle + .departure-row::before`.

Reply:
Sounds good - removed the <div> for <> to keep it flat, now it's looking like header, subtitle, departure, departure, etc.